app/js/uiConfig.js

```diff
@@ -8,6 +8,10 @@
 const defaultConfig = {
 	homePlugin: 'Overview',
 	siadAddress: 'http://localhost:9980',
+	width: 800,
+	height: 600,
```

Comment:
we definitely got a fair amount of complaints about the default size of the UI, people wanted it smaller